README.md

```diff
@@ -11,29 +11,77 @@ The Readme is still a work in progress. It actually won't get you very far, beca
 
 ### Setup
 1. Clone the repo.
-
     ```shell
     git clone https://github.com/silcam/dulu.git
+    ```
 
-2. Create databases dulu_dev and dulu_test. See database.yml for the username and password to use.
+1. Create databases dulu_dev and dulu_test. See database.yml for the username and password to use.
+   
+    ```shell
+    rails db:create
+    ```
+   
+   Note: if you also need to drop the db, you can first run
 
-3. Install bundler if you don't already have it
+    ```shell
+    rails db:drop (if necessary)
+    ```
+
+1. Install bundler if you don't already have it
 
     ```shell
     gem install bundler
+    ```
 
-4. Install the necessary gems
+1. Install the necessary gems
 
     ```shell
-    cd dulu
     bundle install
+    ```
+
+1. yarn install
 
-5. Initialize the development database
+    ```shell
+    yarn install
+    ````
+
+   If you don't have it, install [yarn](https://classic.yarnpkg.com/) first.
+
+1. Initialize the development database by loading the schema
 
     ```shell
     rails db:schema:load
+    ```
+
+### Running the Tests
+
+If you just want to run the tests, you can after a few more steps. The test database does not need to be seeded.
 
-6. Start the server. Access Dulu at http://localhost:3000
+1. In `secrets.yml`, add Rails.application.secrets.gmail_username, set to something fake.
 
+1. Run tests.
+
+   See the definitions in `package.json` about what to do, but for example you can run `yarn test:most` to
+    test most things. (You may also need to install [cypress](https://www.cypress.io/) to do that.)
+
+### Starting the Server
+
+If you want to start the server to run this in a web browser, you need a few more steps.
+
+1. You need to set up an omni auth config in `./config/initializers/omniauth.rb`. You can see the [Omniauth](https://github.com/omniauth/omniauth) documentation for how to do this.
+
+1. You need to seed the database.
+    
     ```shell
-    rails s
\ No newline at end of file
+    rails db:seed RAILS_ENV=development
+    ```
+
+    This will insert some default data and create an admin user for you. Use one of your Google accounts.
+    
+1. Start the Server
+
+    ```shell
+    rails s
```

Comment:
I think `foreman start` would be better. I don't think you can run the app in dev without webpack-dev-server running. I could be wrong though...

Reply:
That's how I started mine and it came up, but I'll change it to use foreman if that's preferred.